4.analyze-components/README.md

```diff
@@ -0,0 +1,36 @@
+# Analyzing Compressed Gene Expression Sample Outputs
+
+**Gregory Way 2018**
+
+Samples within each dataset are compressed with dimensionality reduction algorithms.
+When these samples pass through the bottleneck, they are variably activated by each latent space feature.
+The activations can represent biological or technical patterns present in a specific quantity for the given sample and feature.
+
+The samples are also reconstructed back into the original input gene expression dimensions.
+In this module, we explore the ability of compression models to reconstruct and capture variation in input samples.
```

Comment:
What do you mean by "variation in the input samples" ?  More specifically, how are you measuring the variation in the input samples?  Do you mean variance? Do you mean different datasets?

I don't see an analysis that is looking at the ability of the compression models capturing variation in the input samples?

Reply:
Yeah, nice catch. I don't think I'm expressing what I mean well with "variation in input samples". I think what I really mean is ability to "capture signals present in input samples"